py-polars/src/lib.rs

```diff
@@ -594,6 +594,16 @@ fn set_float_fmt(fmt: &str) -> PyResult<()> {
     Ok(())
 }
 
+#[pyfunction]
+fn get_float_fmt() -> PyResult<String> {
+    use polars_core::fmt::{get_float_fmt, FloatFmt};
+    let strfmt = match get_float_fmt() {
+        FloatFmt::Full => "full",
+        FloatFmt::Mixed => "mixed",
+    };
```

Comment:
If you check `conversion.rs` you can see the default way of parsing string arguments into an enum. I'd recommend following that approach, it will result in a nice ValueError for wrong input.

Reply:
This is the other way round - it's going from a Rust Enum (that cannot have an incorrect value) to string ;)

Reply:
Oooh, right. Nevermind 🙃